_pages/dev/routes/external-routes.md

```diff
@@ -12,128 +12,128 @@ feature:
 
-Thanks to *external routes* feature of Spartacus, some routes can be loaded from backend or even redirected to a different domain.
+***
 
-## How to use Spartacus and other storefront systems at the same domain
+**Table of Contents**
 
-The URL patterns need to be defined to distinguish the storefront systems. Those patterns should be applied in 3 places: *backend server*, *Spartacus* configuration and *Angular service worker* (when PWA is enabled), because of the following reasons:
+- This will become a table of contents (this text will be scrapped).
+{:toc}
 
-1. When accessing a deep link, the *backend server* should:
-    - serve Spartacus; **or**
-    - serve other storefront's page
-2. When navigating by the Angular's `routerLink`, *Spartacus* should:
-    - activate a SPA route; **or**
-    - full (re)load a page
-3. When full (re)loading page, *Angular service worker* should:
-    - intercept navigation request and return cached `index.html` of SPA; **or**
-    - bypass the cache - to let the backend serve the response
+***
 
-### Configuration of backend server
+## Using Spartacus and Another System to Run a Single Storefront
 
-The configuration of URL patterns for backend server depends on the used backend technology. Please refer to its documentation.
+To run a storefront at a single domain using Spartacus along with another system, you define URL patterns to distinguish between the two storefront systems. These patterns should be applied in the back end server, in the Spartacus configuration, and in the Angular service worker (when PWA is enabled). The patterns should be applied as follows:
 
-### Configuration of Spartacus
+- When accessing a deep link, the back end server should serve a Spartacus view, or it should serve a page from the other storefront system.
 
-Provide the configuration with URL patterns for internal routes, i.e. by importing `ConfigModule.withConfig()`. 
+- When using the Angular `routerLink` to navigate, Spartacus should activate a SPA route, or it should fully load (or reload) the page.
 
-> The URL patterns use a limited glob format:
->   * `**` matches 0 or more path segments
->   * `*` matches 0 or more characters excluding `/`
->   * `?` matches exactly one character excluding `/` 
->   * The `!` prefix marks the pattern as being negative, meaning that only URLs that don't match the pattern will be included
+- When fully loading (or reloading) a page, the Angular service worker should intercept the navigation request and return the cached `index.html` of the single-page application, or it should bypass the cache so that the back end can serve the response.
```

Comment:
```suggestion
- In case the PWA is enabled, the Angular service worker intercepts the navigation request. When fully loading (or reloading) a pageit should return the cached `index.html` of the single-page application, or it should bypass the cache so that the back end can serve the response.
```

Reply:
👍 🙂